services/ux-backend/handlers/storages/handler.go

```diff
@@ -0,0 +1,143 @@
+package storages
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"k8s.io/klog/v2"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	deploymentTypeInternal = "internal"
+	deploymentTypeExternal = "external"
```

Comment:
We already have deploymentType defined in [storagecluster/odfInfo](https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/odfinfoconfig.go#L26-L27), let's move it to the utils package and then use it instead of having new constants

Reply:
`deploymentType` Is not something that's going to change even if they do, they have to be changed in both places. They operate independently, but share the same codebase. I think it would be good not to define them again and move them to the utils package